src/Npgsql/PostgresVariables.cs

```diff
@@ -0,0 +1,26 @@
+using System;
+using System.IO;
+using Npgsql.Util;
+
+namespace Npgsql
+{
+    static class PostgresVariables
+    {
+        public static string? User => Environment.UserName ?? Environment.GetEnvironmentVariable("PGUSER");
```

Comment:
I know this matches the previous behavior, but shouldn't the order be flipped to match the `libpq` behavior?

Reply:
It's a bug. Fixed.

Reply:
It does break (by correcting) previous behavior though, right? If so, we should add a warning in the docs/release notes so we don't catch anyone off-guard.